src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/FeatureChainingWfsTest.java

```diff
@@ -408,6 +412,63 @@ public void testDescribeFeatureTypeNoTypes() {
         assertXpathCount(0, "//xsd:element", doc);
     }
 
+    /**
+     * Tests that WFS schema is never imported in a DescribeFeatureType response.
+     *
+     * <p>
+     * <strong>Remarks:</strong> this test only targets WFS 2.0, as 1.1.0 is sufficiently covered
+     * by other tests. 
+     * </p>
+     *
+     * @throws Exception
```

Comment:
Please add a description to the exception tag or remove it, we must make GeoServer JavaDoc clean as possible, otherwise we end up with 1000 JavaDoc warnings who may hide important JavaDoc issues.